### ISSUE TITLE
Add/Edit Event: Image url and on-click link are not saved

### ISSUE THREAD

**Issue by antos-k** [diglabby/mapa#162]
1. Open map
2. Press 'Add new entry' (or edit existing event)
3. Select 'Event' category as Event
4. Fill all fields correctly
5. Fill Image field with correct url of any image, for example, https://sun9-55.userapi.com/c857736/v857736486/4458a/xmgrrpcq_Ek.jpg
6. Fill other fields correctly
7. Press Save

**Results:** Image and link are not saved (we can check it if open this event for editing). At this time for Initiatives all saved correctly.

**Expected:** Image link and url are saved successfully

![Selection_019](https://user-images.githubusercontent.com/53947213/63182186-a1768700-c05a-11e9-8158-085774c3826c.png)

**Comment by PolinaSam** [comment]
@antos-k, I tried to add an event and incountered the following error 
![mistake](https://user-images.githubusercontent.com/56090664/71021592-5d2ded00-2107-11ea-973e-2299430929fd.png)

**Comment by rizomaa** [comment]
@Anthony-Hoopkins does it still have importance now?

**Comment by rizomaa** [comment]
Add tests
1. Different browsers (browser stack)
2. Tablet or mobile.
3. Social networks (inst, vk, fb)

**Comment by ArtyomKr** [comment]
I tried to test it, and it's a bigger issue: I can't add any events at all. It seem that there's some issues with CORS headers at https://api.ofdb.io/v0/events. Maybe https://mapa.falanster.by/ doesn't have access to this url.
![Annotation1](https://user-images.githubusercontent.com/53486579/83943633-63736b00-a806-11ea-90c2-001208e57d7c.png)
![Annotation2](https://user-images.githubusercontent.com/53486579/83943642-6ff7c380-a806-11ea-8dd0-bf9dbd6de105.png)
![Annotation 2020-06-05 173008](https://user-images.githubusercontent.com/53486579/83943665-9e759e80-a806-11ea-9470-c5c5600fbd48.png)

**Comment by ArtyomKr** [comment]
@rizomaa I need some input from someone who knows about project back-end

**Comment by rizomaa** [comment]
@ArtyomKr hello,

Thanks for the request. I send the issue to the german developer, when they share link you will be invited.

Do you try to add event online or on you local frontend installation?

**Comment by ArtyomKr** [comment]
@rizomaa  I tried both and the result is the same.
![image](https://user-images.githubusercontent.com/53486579/84387030-09eeb000-abfb-11ea-825d-78ae6c4787be.png)

**Comment by wellemut** [comment]
It is really strange.
In kvm it works perfectly: https://kartevonmorgen.org/#/?entry=62b2d765dd5b4e1b9091b3d59a3a2507&zoom=15.00
Same entry in map does not work: https://mapa.falanster.by/#/?entry=62b2d765dd5b4e1b9091b3d59a3a2507&zoom=15.00

@rizomaa Is is definitly a frontend problem. I suggest you compare the frontendcode of kvm

**Comment by rizomaa** [comment]
@wellemut thanks.

Could you provide your URL.js ? It helps us to understand configuration

**Comment by navid-kalaei** [comment]
I tried to reproduce the error but failed and the project is working flawlessly. Could you please try to check if the issue still exists? If so could I kindly give me a clue how should I reproduce that? Maybe a short video could help. Thanks in advance.

**Comment by ArtyomKr** [comment]
@navid-kalaei 
Thanks for your input. It still exists.
1. Go to https://mapa.falanster.by/#map .
2. Click "add new entry".
3. Select event ("падзея").
4. Fill the necessary fields.
5. Click add.
![image](https://user-images.githubusercontent.com/53486579/84565851-eea7b000-ad74-11ea-9c58-31d45d17f650.png)
It happens every time for me

**Comment by navid-kalaei** [comment]
@ArtyomKr you are very welcome :+1: 

The server that serves your front-end should be allowed by our backend. I'll check that with @flosse and @uklotzde to find the solution.

**More details**:
- `Access to XMLHttpRequest at 'https://api.ofdb.io/v0/events' from origin 'https://mapa.falanster.by' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.`
- `POST https://api.ofdb.io/v0/events net::ERR_FAILED`

**Comment by rizomaa** [comment]
@ArtyomKr  please look here, maybe it helps you also
https://github.com/diglabby/mapa/pull/368

**Comment by rizomaa** [comment]
from telegram
"
Navid Kalaei, [13.06.20 05:56]
Rizoma we had an issue about six months ago or so that the redux debugger extension had made karte von morgen halt. please update that script asap. clients with redux extention enabled are not able to open mapa. the fixation is currently deployed on our repository.

Navid Kalaei, [13.06.20 06:09]
Rizoma I checked that. Strangely I could not reproduce the error you have shown and everything is working perfectly. could you please retry to see if you still have the error?
"

**Comment by rizomaa** [comment]
@ArtyomKr any progress here?

**Comment by ArtyomKr** [comment]
@uklotzde Hello, maybe you know something about this problem?  I'm talking about https://github.com/diglabby/mapa/issues/162#issuecomment-640048844 .

**Comment by rizomaa** [comment]
@ArtyomKr If the Adiing initiative works but not event. Did you compare code in both places?
Because from developer perspective event and initiative are just different tags here.

**Comment by ArtyomKr** [comment]
@rizomaa Yes, I compared the code for both and it looks similar. This issue is now my priority so I'll dig deeper into this.

**Comment by rizomaa** [comment]
@navid-kalaei cuould you share us you config for frontend?

@ArtyomKr ok, any thoughts and progress put here)

**Comment by ArtyomKr** [comment]
I tried to reproduce original issue, but I didn't encounter any problems. I think it's closed by #397 .

**Comment by rizomaa** [comment]
@ArtyomKr thanks, if you checked something and consider as a done please move to "Review in progress" column
